src/portal/templates/portal/base.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <meta name="description" content="">
+        <meta name="author" content="">
+        <link rel="icon" href="../../favicon.ico">
+
+        {% load staticfiles %}
+        <title>APOF</title>
+        <!-- Bootstrap core CSS -->
+        <link rel="stylesheet" href="{% static "portal/css/bootstrap.min.css" %}">
+
+        <!-- Custom styles for this template -->
+        <link rel="stylesheet" href="{% static "portal/css/style.css" %}">
+        <link rel="stylesheet" href="{% static "portal/css/topbar.css" %}">
+        <link rel="stylesheet" href="{% static "portal/css/sidebar.css" %}">
+        <link rel="stylesheet" href="{% static "portal/css/mobile.css" %}">
```

Comment:
Please be consistent. When you use `{% extends 'portal/base.html' %}` or `{% include 'portal/sidebar.html' %}` you use single quotes, but there you use double quotes.